frontend/public/components/nav/admin-nav.tsx

```diff
@@ -146,9 +148,18 @@ const AdminNav = () => (
 
     <NavSection title="Compute" required={FLAGS.CAN_LIST_NODE}>
       <ResourceClusterLink resource="nodes" name="Nodes" />
-      <ResourceNSLink resource={referenceForModel(MachineModel)} name="Machines" required={FLAGS.CLUSTER_API} />
-      <ResourceNSLink resource={referenceForModel(MachineSetModel)} name="Machine Sets" required={FLAGS.CLUSTER_API} />
-      <ResourceNSLink resource={referenceForModel(MachineAutoscalerModel)} name="Machine Autoscalers" required={FLAGS.MACHINE_AUTOSCALER} />
+      <HrefLink href={formatNamespacedRouteForResource(referenceForModel(MachineModel), 'openshift-machine-api')}
+        name="Machines"
+        required={FLAGS.CLUSTER_API}
+        startsWith={machineStartsWith} />
+      <HrefLink href={formatNamespacedRouteForResource(referenceForModel(MachineSetModel), 'openshift-machine-api')}
+        name="Machine Sets"
+        required={FLAGS.CLUSTER_API}
+        startsWith={machineStartsWith} />
```

Comment:
I don't think these items can have the same `startsWith`. Won't they both be highlighted?